packaging/snapd.mk

```diff
@@ -50,14 +50,14 @@ snap_mount_dir = /snap
 endif
 
 # The list of go binaries we are expected to build.
-go_binaries = snap snapctl snap-seccomp snap-update-ns snap-exec snapd snap-recovery
```

Comment:
Note, you can still keep this part but expand the part below that removes non-core binaries.